Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'cocoapods'
+gem 'cocoapods-fix-react-native', :git => 'git@github.com:ZachOrr/cocoapods-fix-react-native.git', :branch => 'zorr/fix-0.58.6'
```

Comment:
The real fix here is get the regular gem working and then use that. I'm running in to errors - might ask a Cocoapods dude at some point for some help.